### DISCUSSION TITLE
Extremely high error of forces

### DISCUSSION THREAD

**Discussion by phys-chem** [ACEsuit/mace#350] [Q&A] [answered]
The initial: Epoch 0: loss=0.0261, RMSE_E_per_atom=51.1 meV, RMSE_F=8.5 meV / A
final: Epoch 796: loss=0.1917, RMSE_E_per_atom=13.9 meV, RMSE_F=2526.2 meV / A

The energy RMSE decreases but force RMSE increases dramatically. I want to know how to solve it. Thank you very much!
Here is the training script:
```
mace_run_train \
        --name="MOF"\
        --train_file="../../MOF_0.1_train.xyz" \
    --valid_file="../../MOF_0.1_test.xyz" \
    --E0s="average" \
    --model="MACE" \
    --num_interactions=2 \
    --num_channels=64 \
    --max_L=0 \
    --correlation=3 \
    --r_max=6.0 \
    --forces_weight=10000 \
    --energy_weight=10 \
        --virials_weight=0.0 \
        --stress_weight=0.0 \
         --swa_forces_weight=10 \
    --swa_energy_weight=1000 \
        --batch_size=2 \
    --valid_batch_size=4 \
    --max_num_epochs=800 \
    --start_swa=400 \
    --scheduler_patience=15 \
    --patience=30 \
    --eval_interval=4 \
    --ema \
    --swa \
    --error_table='PerAtomRMSE' \
    --default_dtype="float64"\
    --device=cuda \
    --seed=123 \
    --restart_latest \
    --save_cpu \

```
Training logs are attached:
[MOF_run-123.log](https://github.com/ACEsuit/mace/files/14672547/MOF_run-123.log)

**Comment by ilyes319** [comment, marked as answer]
Can you check if you gave the correct force key in your xyz. I would guess your forces are all zeros atm.

**Reply by phys-chem** [reply]
Thank you! The data format is "Properties=species:S:1:pos:R:3:force:R:3". After setting the force key, it has been solved.
